redis-persistence/src/main/java/com/netflix/conductor/redis/dynoqueue/RedisPinger.java

```diff
@@ -26,6 +26,46 @@ public class RedisPinger {
     private static final Logger log = LoggerFactory.getLogger(RedisPinger.class);
     private static final int CONNECTION_TIMEOUT = 3000;
 
+    private static int MAX_RETRY_COUNT = 3;
+    private static int RETRY_INTERVAL = 1000;
```

Comment:
retry interval of 1 second may be very short. How about using exponential backoff? Something like
1st attempt - wait for 2s
2nd attempt - wait for 3s
3rd attempt - wait for 4.5s.
Example - https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/backoff/ExponentialBackOff.html

Reply:
its implemented